src/pages/item/ItemInfo.jsx

```diff
@@ -168,30 +178,47 @@ const ItemInfo = () => {
         <div className="item-price-card">
           <h4>price</h4>
           <div className="item-addcard-price">
-            <h2>{price} $</h2>
+            <h2>$ {price} </h2>
             <div className="add-card-count">
-              <span onClick={removingProductCount}>
+              <span
+                onClick={() => {
+                  removingProductCount();
```

Comment:
`removing`? `removeProductCount` is not better? it is doing a job, removing is the ongoing thing which is not finished yet. the function name should be an actual verb